.teamcity/performance-test-durations.json

```diff
@@ -639,7 +639,7 @@
     "linux" : 204
   } ]
 }, {
-  "scenario" : "org.gradle.performance.regression.nativeplatform.NativeBuildDependentsPerformanceTest.run libA0:buildDependentsLibA0",
+  "scenario" : "org.gradle.performance.regression.nativeplatform.NativeBuildDependentsPerformanceTest.run libA0:buildDependentsLibA00",
```

Comment:
As discussed, this is worth adding in the upgrade guide

Reply:
https://github.com/gradle/gradle/pull/30538